server/config/config.go

```diff
@@ -162,6 +162,8 @@ type Config struct {
 	Dashboard DashboardConfig `toml:"dashboard" json:"dashboard"`
 
 	ReplicationMode ReplicationModeConfig `toml:"replication-mode" json:"replication-mode"`
+
+	DisableServiceMiddleware bool
```

Comment:
Once upon a time, PM said we should not use configurations starts with `Disable` but only use that starts with `Enable`.

Reply:
Better change to `EnableServiceMiddleware`, and can this be dynamically turned on and off?